server/modules/watchers/publishFields.ts

```diff
@@ -64,6 +65,7 @@ export const roomFields = {
 	prid: 1,
 	avatarETag: 1,
 	usersCount: 1,
+	canPlaceChatOnHold: 1,
```

Comment:
This should not be a field.

Reply:
Removed this property as its no longer required